docs-source/content/faq/oci-lb.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Using an OCI Load Balancer"
+date: 2019-09-25T12:41:38-04:00
+draft: false
+---
+
+If you are running your Kubernetes cluster on Oracle Container Engine 
+for Kubernetes (commonly known as OKE), you can have OCI automatically
+provision load balancers for you by creating a `Service` of type
+`LoadBalancer` instead of (or in addition to) installing an
+ingress controller like Traefik or Voyager.
+
+OKE Kubernetes worker nodes normally do not have public IP addresses.
+This means that the `NodePort` services created by the operator are 
+not usable, since they would expose ports on the worker node's public IP
+address.  Instead, you can use an OCI Load Balancer to provide access
+to services running in OKE.
+
+{{% notice note %}}
+It is also also possible, if desirable, to have an OCI Load Balancer route
+traffic to an ingress controller running inside the Kubernetes cluster
+and have that ingress controller in turn route traffic to services in the
+cluster.
+{{% /notice %}}
+
+
+### Requesting an OCI load balancer
+
+When your domain is created by the operator, a number of Kubernetes 
+Services are created by the operator, including one for the WebLogic 
+Administration server and one for each managed server and cluster.
+
+In the example below, there is a domain called `bobs-bookstore` in the
+`bob` namespace.  This domain has a cluster called `cluster-1` which 
+exposes traffic on port `31111`.
+
+The Kubernetes YAML file below defines a new `Service` in the same
+namespace.  The `selector` targets all of the pods in this namespace
+which are part of the cluster `cluster-1` using the annotations that 
```

Comment:
namespace which are part of the cluster `cluster-1` -> namespace, which are part of the cluster, `cluster-1`,

Reply:
i think without commas is actually right, adding commas changes the meaning i think

Reply:
You would know better than I (wasn't sure if it was a restrictive or non-restrictive clause), however, I do think you should put a comma before and after `cluster-1`. It would help the readability and be grammatically correct.

Reply:
done